component.json

```diff
@@ -59,6 +101,47 @@
         "out": {}
       }
     },
+    "read_action_v2": {
+      "main": "./lib/triggers/read_v2.js",
+      "title": "Read CSV attachment v2",
+      "fields": {
+        "url": {
+          "label": "CSV URL",
+          "required": false,
+          "placeholder": "http://my-url.com/foo.csv",
+          "note": "We will fetch this URL and parse it as CSV file, leave the field blank if you expect CSV attachment from previous step",
```

Comment:
I would skip this and make the URL required.

Reply:
Done